crates/ruff_linter/src/rules/flake8_pie/snapshots/ruff_linter__rules__flake8_pie__tests__PIE804_PIE804.py.snap

```diff
@@ -119,4 +120,19 @@ PIE804.py:11:1: PIE804 [*] Unnecessary `dict` kwargs
 13 13 | 
 14 14 | foo(**{**data, "foo": "buzz"})
 
+PIE804.py:25:1: PIE804 [*] Unnecessary `dict` kwargs
+   |
+24 | # Duplicated key names wont be fixed to avoid syntax error.
+25 | abc(**{'a': b}, **{'a': c})  # PIE804
+   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^ PIE804
+   |
+   = help: Remove unnecessary kwargs
+
+ℹ Safe fix
+22 22 | abc(**{"for": 3})
+23 23 | 
+24 24 | # Duplicated key names wont be fixed to avoid syntax error.
+25    |-abc(**{'a': b}, **{'a': c})  # PIE804
+   25 |+abc(a=b, **{'a': c})  # PIE804
```

Comment:
Sorry about the delay in the review here!

Should we even offer to fix the first one if the whole thing is invalid?

Reply:
Fix does have exactly the same behavior and doesn't change the code.
Also, only syntax errors could be unsafe fix, so it doesn't even have need to be unsafe (discussed above)

If we want to change it to unsafe Or sometimes fixable violation, we should also consider this:
```py
abc(**a, **{'b': 0})
```
where `a` may have `b` key inside.

Reply:
I think it need another rule when there is multiple unknown keyword unpacking on calling callables.
examples:
```
abc(**a, **b)
abc(**a, **{'b': 0})
abc(a=2, **b)
```
all of these potentially can be runtime error.

Reply:
> Fix does have exactly the same behavior and doesn't change the code.

Yes but the "same behavior" here is a runtime error. I'm worried it will be confusing to users to fix one violation of this then not the second. We cannot fix the second violation because we don't want to introduce a syntax error. It might be clearer to the user not suggesting _any_ fixes when we can tell the code has a runtime error.